backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/FileService.java

```diff
@@ -122,13 +122,15 @@ public String[] parseLinesFromFile(String fileName, String fileDirectory) {
      *
      * @param id Canvas user id associated with the directory
      */
-    public void deleteDirectory(String id) {
+    public boolean deleteDirectory(String id) {
```

Comment:
I pulled and tested this from my end and it seems to return true even if the directory doesn't exist. How about we just create another function in FileService to return whether or not the directory exists using [`exists()` from the File library](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#exists()) and use the returned boolean value for the deletion response instead?

Reply:
Good catch. Will do and hopefully not create a new branch in the process